frontend/src/lib/forms/Select.svelte

```diff
@@ -1,23 +1,16 @@
 <script lang="ts">
+  import FormField from './FormField.svelte';
+  import { randomFieldId } from './utils';
+
   export let label: string;
-  export let id: string;
-  export let autofocus = false;
-  export let error: string | string[] = '';
   export let value: string;
+  export let id = randomFieldId();
+  export let autofocus = false;
+  export let error: string | string[] | undefined = undefined;
 </script>
 
-<label for={id} class="label">
-  <span class="label-text">
-    {label}
-  </span>
-</label>
-<!-- svelte-ignore a11y-autofocus -->
-<select {autofocus} bind:value {id} class="select select-bordered">
-  <slot />
-</select>
-
-{#if error}
-  <label for={id} class="label">
-    <span class="label-text-alt text-error mb-2">{typeof error === 'string' ? error : error.join(', ')}</span>
-  </label>
-{/if}
+<FormField {id} {label} {error} {autofocus}>
+  <select bind:value {id} class="select select-bordered">
```

Comment:
don't we still want an autofocus attribute on the field? that way the browser can autofocus for us which is faster than waiting for svelte to focus it?

Reply:
Good point 👍

Reply:
I [pushed the attributes](https://github.com/sillsdev/languageforge-lexbox/commit/bf566f22e5bd2dae2b7db04260b765c860074b49) directly to develop.